Library/Formula/cmake.rb

```diff
@@ -34,7 +34,6 @@ class Cmake < Formula
   option "without-docs", "Don't build man pages"
   depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
   depends_on "xz" # For LZMA
-  depends_on "qt" => :optional
```

Comment:
Sure, will do. Do you want to note the willingness to consider a ` cmake-gui ` formula at some point, or are we completely cool with scrapping the GUI support altogether?